src/client/js/WebGME.js

```diff
@@ -89,8 +89,10 @@ define([
             // URL query has higher priority than the config.
             if ((initialThingsToDo.projectToLoad || initialThingsToDo.createNewProject) === false) {
                 initialThingsToDo.projectToLoad = gmeConfig.client.defaultProject.name;
-                initialThingsToDo.branchToLoad = initialThingsToDo.branchToLoad || gmeConfig.client.defaultProject.branch;
-                initialThingsToDo.objectToLoad = initialThingsToDo.objectToLoad || gmeConfig.client.defaultProject.node;
+                initialThingsToDo.branchToLoad = initialThingsToDo.branchToLoad ||
+                    gmeConfig.client.defaultProject.branch;
+                initialThingsToDo.objectToLoad = initialThingsToDo.objectToLoad ||
+                    gmeConfig.client.defaultProject.node || initialThingsToDo.objectToLoad; // i.e. the root-node.
```

Comment:
exactly.
I was not saying that it not fixed the original issue. I was pointing out, that it still not the best we can come up with.
so if we think about the scenario I wrote, then we can simplify the question on whether the config has higher priority than the url parameters.
I think the url parameters has the highest priority which is why I proposed the two alternative options to fix the issue as well as get rid of another.